src/article.py

```diff
@@ -66,7 +65,7 @@ class Article:
     """
 
     # pylint: disable=too-many-instance-attributes
-    # Eight attributes are ok here.
+    # Eight attributes are fine here.
```

Comment:
One way how you can solve this problem is by bundling the data.

```
from pydantic import BaseModel

class DownloadRequest(BaseModel):
    title: str
    authors: List[str]
    ...

def __init__(self, request: DownloadRequest):
  ...
```

Otherwise please at least use `*` to force keyword arguments. Noone remembers if abstract was the 7th or 6th argument of this function.

Reply:
Thanks for the suggestion! I am thinking about shrinking the Article class and include DownloadRequest or something similar to be more inline with S.O.L.I.D. & Co. ;)